Validation/RecoTau/src/TauTagValidation.cc

```diff
@@ -153,12 +161,20 @@ void TauTagValidation::bookHistograms(DQMStore::IBooker & ibooker, edm::Run cons
   phiTauVisibleMap.insert( std::make_pair(TauProducer_+"Matched" ,phiTemp));
   pileupTauVisibleMap.insert( std::make_pair(TauProducer_+"Matched" ,pileupTemp));
 
-  for ( std::vector< edm::ParameterSet >::iterator it = discriminators_.begin(); it!= discriminators_.end();  it++)
+  int j = 0;
+  for ( std::vector< edm::ParameterSet >::iterator it = discriminators_.begin(); it!= discriminators_.end();  it++, j++)
```

Comment:
In principle you can modernize the code using "loops in ranges" (here and in other places inside this and other modified plugins). It is well received during central revisions.